packit_service/service/api/webhooks.py

```diff
@@ -344,3 +353,119 @@ def interested():
 
         logger.debug(f"{event_type} {' (not interested)' if not _interested else ''}")
         return _interested
+
+
+forgejo_webhook_calls = Counter(
+    "forgejo_webhook_calls",
+    "Number of times the Forgejo webhook is called",
+    ["result", "process_id"],
+)
+
+
+@ns.route("/forgejo")
+class ForgejoWebhook(Resource):
+    @ns.response(HTTPStatus.OK.value, "Webhook accepted, returning reply")
+    @ns.response(
+        HTTPStatus.ACCEPTED.value,
+        "Webhook accepted, request is being processed",
+    )
+    @ns.response(HTTPStatus.BAD_REQUEST.value, "Bad request data")
+    @ns.response(HTTPStatus.UNAUTHORIZED.value, "X-Forgejo-Signature validation failed")
+    @ns.expect(ping_payload_forgejo)
+    def post(self):
```

Comment:
I would probably add the docstring.

```suggestion
    def post(self):
    """
    A webhook used by Packit-as-a-Service Forgejo hook.
    """
```